src/modules/apps/link.ts

```diff
@@ -29,7 +31,71 @@ const UPDATE_SIGN = chalk.blue('U')
 const stabilityThreshold = process.platform === 'darwin' ? 100 : 200
 const AVAILABILITY_TIMEOUT = 1000
 const N_HOSTS = 3
+const builderHubInjectedDepsTimeout = 2000  // 2 seconds
+const yarn = join(__dirname, '../../../node_modules/yarn/bin/yarn.js install --force')
```

Comment:
Not sure if the way `builder-hub` calls and pass arguments to `yarn` looks a little bit better: https://github.com/vtex/builder-hub/blob/master/service/build/yarn.ts#L76

Reply:
The `execSync` function does not support an array of options for the command. That said, I made the `runYarn` function a little cleaner now.